src/settings/toml/manifest.rs

```diff
@@ -349,6 +349,12 @@ impl Manifest {
                         if build.command.is_none() {
                             anyhow::bail!(error_message)
                         }
+                        if matches!(build.upload, UploadFormat::Modules { .. }) {
```

Comment:
This might not be true as of later tonight. We're landing https://github.com/cloudflare/wrangler/pull/2126, and making a release of https://github.com/cloudflare/kv-asset-handler and will update the docs showing hwo it works with es modules.